index.html

```diff
@@ -288,6 +288,24 @@ <h2>About me</h2>
             </li>
           </ul>
         </div>
+        <!-- contact form section -->
```

Comment:
- [ ] For a more clean and readable codebase, remove inline comments.

Reply:
- Alright noted 👍